server/db/models/user.js

```diff
@@ -26,6 +26,24 @@ const User = db.define('user', {
   },
   googleId: {
     type: Sequelize.STRING
+  },
+  firstName: {
+    type: Sequelize.STRING,
+    allowNull: false,
+    validate: {
+      notEmpty: true
+    }
+  },
+  lastName: {
+    type: Sequelize.STRING,
+    allowNull: false,
+    validate: {
+      notEmpty: true
+    }
+  },
+  admin: {
```

Comment:
Should also not be null or empty

Reply:
If it is null or empty, the default value will set it to false.